New site launch

ewels opened the pull request:
Do not merge until we are ready to go live with the new site.

ewels commented:
Apologies if some of this is underway, but figured it was easiest to make a list on the PR. I'm using checkboxes, hopefully you can tick them off as they're done ✅ 

You're working on Program and Speakers currently, so I've not looked at them 🙈 😁 

## Header / Footer

1. [ ] Footer should be fixed to the bottom of the page, see this Figma preview:

https://user-images.githubusercontent.com/465550/177201751-99ffd4bd-7b18-4962-8036-22bf8f31c683.mov

2. [ ] Social icons in bottom right of the footer are missing
3. [ ] `© 2022 Seqera Labs` should link to `https://seqera.io`
4. [ ] Header links should have active state (green) when on that page.
5. [ ] No navigation present on mobile

## Homepage

1. [ ] No background gradient
2. [ ] Scrolling _"Call for abstracts now open."_ text should link to `call-for-abstracts/`
3. [ ] Nav should be transparent on this page only
4. [ ] Key Information > Venue: Missing map pin icon
5. [ ] Key Information > Streaming: Old incorrect content + missing icons
6. [ ] Key Information: Can we drop the `1rem` top-margin for the bullet points?

## Call for abstracts

1. [ ] Scrolling _"Call for abstracts now open."_ text needs gradient background (no link, as this is the target page)
2. [ ] Images have no rounded corners
3. [ ] Background for images have too much rounding - `12px` but should be `4px`
4. [ ] Images extend beyond page width, need overflow hiding (right hand side only, left is fine)
    <img width="300" alt="image" src="https://user-images.githubusercontent.com/465550/177202743-09a26f0a-3e46-447b-b578-dc941380aaf8.png">
5. [ ] _"Apply now"_ button should have external-link icon added

## Call for abstracts > Instructions

1. [ ] Back icon is missing <kbd><</kbd> icon
2. [ ] Both panels missing the _"Apply now"_ text link at the bottom (instructions PDF not ready so that link in Figma can be ignored)
    * _"Apply now"_ text link should link back to `call-for-abstracts/`
4. [ ] _"View Abstracts"_ button - I expected some hover state, maybe green background?
5. [ ] I'm pretty sure that the title should be white. It's green in Figma, I've left a comment to ask, but for now change it to white 👍🏻 

## Venue

1. [ ] Image overlaps right hand edge of screen, as in _Call for abstracts_
2. [ ] Scrolling _"Call for abstracts now open."_ needs gradient background and link to `call-for-abstracts/`
3. [ ] Google Maps API needs fixing
4. [ ] Missing map pin icon after address line

## Register

1. [ ] External link icon for the _Register_ button
2. [ ] Typo in fees > virtual: `aNextflow` should be `Nextflow` (was in Figma too, just corrected there)

christopher-hakkaart commented:
Looks really good!

Only issue I found
- [ ] Home page has extra scroll space. Video attached.

https://user-images.githubusercontent.com/76951771/177349613-dca8a9d8-93a2-441c-a87c-247028bc9ca9.mov

ewels commented on src/layout/Footer.jsx:
```diff
@@ -0,0 +1,51 @@
+            </div>
+            <div className="flex items-center md:justify-end">
+              <div className="mr-4">
+                <Link to="http://twitter.com/hashtag/NextflowSummit" noBorder>
```
This is bizarre and I don't understand why, but without the `?src` query string I get no results on the twitter link. Adding this seems to fix it.. 🤷🏻‍♂️ 

```suggestion
                <Link to="https://twitter.com/hashtag/NextflowSummit?src=hashtag_click" noBorder>
```

ewels commented on src/pages/program/nf-core-hackathon.jsx:
```diff
@@ -0,0 +1,94 @@
+                  The nf-core hackathon will be held before the Nextflow Summit, from 10-12 Oct, 2022.
+                </p>
+                <p className="typo-body mb-4">
+                  <Link to="/" className="text-green-600">
```
```suggestion
                  <Link to="https://nf-co.re/events/2022/hackathon-october-2022" className="text-green-600">
```

ewels commented on src/pages/program/nf-core-hackathon.jsx:
```diff
@@ -0,0 +1,94 @@
+                  to learn more.
+                </p>
+                <p className="typo-body">
+                  <Link to="">
```
```suggestion
                  <Link to="https://goo.gl/maps/K3chvdYLa9BfDpaD9">
```

ewels commented on src/pages/register.jsx:
```diff
@@ -0,0 +1,128 @@
+                  Nextflow Summit 2022
+                </h3>
+                <p className="typo-body mt-6">
+                  Join the nf-core hackathon held in Barcelona and virtually.
```
```suggestion
                  Join the Nextflow Summit held in Barcelona and virtually.
```

ewels reviewed (commented):
Some minor copy / link errors, see suggested changes for fixes 👇🏻

ewels commented:
Updated condensed list of remaining points from https://github.com/nextflow-io/summit-website/pull/1#issuecomment-1174060204:

## Header / Footer

1. [ ] Footer should be sticky, fixed to the bottom of the page
2. [ ] Scrolling _"Call for abstracts now open"_ should also be sticky + fixed to bottom of the page (above footer)
3. [ ] Scrolling _"Call for abstracts now open"_ needs colour gradient background (all pages except homepage)
6. [ ] Homepage -> Key Information: Can we drop the `1rem` top-margin for the bullet points?
5. [ ] External link icons for _"Apply now"_ (call for abstracts page main button) / _"Register"_ button (register main button)

ewels commented on src/pages/program/oct-14.jsx:
```diff
@@ -0,0 +1,178 @@
+
+              <div className="bg-black border border-gray-800 px-4 py-6 lg:p-8 rounded-md shadow-xl mt-4">
+                <p className="typo-intro text-green-600 mb-4">
+                  12 AM (20 min)
```
```suggestion
                  1:20 - 2:20 PM (60 min)
```

ewels commented on src/pages/index.jsx:
```diff
@@ -0,0 +1,205 @@
+                <Accordion.Detail>
+                  <List type="bullet" iconClassName="text-white">
+                    <List.Item className="typo-body">
+                      July 1: Registration for the Nextflow Summit opens
```
```suggestion
                      July 7: Registration for the Nextflow Summit opens
```

ewels commented on src/pages/call-for-abstracts/index.jsx:
```diff
@@ -0,0 +1,140 @@
+            Submit your talk or poster for one of the key themes: Nextflow, community, ecosystem, or software.
+          </p>
+          <p className="typo-body">
+            Now - July 22, 2022: Call for abstracts
```
```suggestion
            June 17 - July 22, 2022: Call for abstracts
```

ewels commented on src/pages/program/oct-14.jsx:
```diff
@@ -0,0 +1,178 @@
+              <div className="bg-black border border-gray-800 px-4 py-6 lg:p-8 rounded-md shadow-xl">
+                <div className="flex items-center justify-between mb-4">
+                  <p className="typo-intro text-green-600">
+                    11 - 12 AM (60 min)
```
```suggestion
                    11 AM - 12 PM (60 min)
```
